mlflow/data/schema.py

```diff
@@ -0,0 +1,82 @@
+from mlflow.exceptions import MlflowException
+from mlflow.types.schema import Schema
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from typing import Dict, Any
+
+
+class TensorDatasetSchema:
```

Comment:
Taken from https://github.com/mlflow/mlflow/blob/a23c759326d4559342d1f64a276e16e8a837aab9/mlflow/models/signature.py#L35 with fields renamed, since `inputs` and `outputs` of `ModelSignature` don't really make sense in the context of datasets.

This is used to package features and targets as a single schema object.